digits/views.py

```diff
@@ -166,6 +166,21 @@ def job_status(job_id):
         result['type'] = job.job_type()
     return json.dumps(result)
 
+@app.route('/job_management', methods=['GET'])
+@autodoc('util')
+def job_management():
+    """
+    Return the jobs management page
+
+    """
+
+    running_datasets    = get_job_list(dataset.DatasetJob, True)
+    running_models      = get_job_list(model.ModelJob, True)
+
+    return flask.render_template('job_management.html',
+                                 running_job = running_datasets + running_models,
+                             )
+
```

Comment:
@jmancewicz did you mean to leave this route in? It throws an error on my machine when I try to access this url:
```
UndefinedError
'running_datasets' is undefined

Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/share/digits/digits/views.py", line 181, in job_management
    running_job = running_datasets + running_models,
  File "/usr/lib/python2.7/dist-packages/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/usr/lib/python2.7/dist-packages/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/usr/lib/python2.7/dist-packages/jinja2/environment.py", line 969, in render
    return self.environment.handle_exception(exc_info, True)
  File "/usr/lib/python2.7/dist-packages/jinja2/environment.py", line 742, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/share/digits/digits/templates/job_management.html", line 204, in top-level template code
    {% block content %}
  File "/usr/share/digits/digits/templates/job_management.html", line 206, in block "content"
    {% set running_jobs = running_datasets + running_models %}
UndefinedError: 'running_datasets' is undefined
```